pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -131,7 +132,7 @@ func (p *provisioningClusterValidator) validateClusterName(request *webhook.Requ
 	if !isValidName(cluster.Name, cluster.Namespace, err == nil) {
 		response.Result = &metav1.Status{
 			Status:  "Failure",
-			Message: "cluster name must be 63 characters or fewer, cannot be \"local\" nor of the form \"c-xxxxx\"",
+			Message: "cluster name must be 63 characters or fewer, cannot be \"local\" nor of the form \"c-xxxxx\", and can only contain lowercase alphanumeric characters or ' - '",
```

Comment:
Should be add something along the lines of "cannot start with `-` to the message?

Reply:
Sure, hopefully the wording i've added doesn't sound odd - it's a long message